docs/trace/customizing-the-sdk/README.md

```diff
@@ -378,17 +378,15 @@ For the below examples imagine an exporter with this constructor:
 ```csharp
 public class MyCustomExporter : BaseExporter<Activity>
 {
-    public MyCustomExporter(
-        ILogger<MyCustomExporter> logger,
```

Comment:
Note: I removed `ILogger` from this example. I was thinking that might lead people into trouble with endless log/export loops. @cijothomas @utpilla Should we add something in extending-the-sdk about how to log safely in an exporter? Maybe we mention that somewhere already I just checked briefly and didn't see anything.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace/extending-the-sdk has note about SuppressInstrumentationScope... That would help, right?

Reply:
To fix the loop, yes, totally helps! But if user was actually trying to write logs into their OTel stream, it will just eat them 😄 What I was thinking was more of an example that showed how to use `EventSource` to do it?